src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -102,6 +112,14 @@ public bool ImportRequestContext()
             return false;
         }
 
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
The batch container is serialized every time it is sent to a consumer.
The sequence token of the batch is required by consumers, so serialization is necessary.